src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -39,13 +98,53 @@ public PlayerEditor() {
 
         add(new JScrollPane(playerInfoList,
                 JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED, JScrollPane.HORIZONTAL_SCROLLBAR_NEVER), "span, grow");
+
+        /*playerInfoList.addListSelectionListener(event -> {
+            if (!event.getValueIsAdjusting()) {
+
+            }
+        });*/
+
+        playerInfoList.addMouseListener(new MouseAdapter() {
```

Comment:
Mouse listener is NOT the tool for the task, you can select and unselect players using the keyboard, and all of this logic will essentially be broken

Please use a listener for the selection model on the list